models/data_quality/intermediate/data_quality__medical_claim_date_trends.sql

```diff
@@ -0,0 +1,195 @@
+        , {{ dbt.concat(["year", dbt.right(dbt.concat(["'0'", "month"]), 2)]) }} as year_month
+        , distinct_count
+    from
+    (
+        select
+            'discharge_date' as date_field
+            , cast({{ date_part("year", "discharge_date") }} as {{ dbt.type_string() }}) as year
+            , cast({{ date_part("month", "discharge_date") }} as {{ dbt.type_string() }}) as month
+            , count(distinct claim_id) as distinct_count
+        from {{ ref('core__medical_claim') }}
+        group by
+            cast({{ date_part("year", "discharge_date") }} as {{ dbt.type_string() }})
+            , cast({{ date_part("month", "discharge_date") }} as {{ dbt.type_string() }})
+    )x
+
+    union all
+
+    select
+        date_field
+        , {{ dbt.concat(["year", dbt.right(dbt.concat(["'0'", "month"]), 2)]) }} as year_month
+        , distinct_count
+    from
+    (
+        select
+            'medical paid_date' as date_field
+            , cast({{ date_part("year", "paid_date") }} as {{ dbt.type_string() }}) as year
+            , cast({{ date_part("month", "paid_date") }} as {{ dbt.type_string() }}) as month
+            , count(distinct claim_id) as distinct_count
+        from {{ ref('core__medical_claim') }}
+        group by
+            cast({{ date_part("year", "paid_date") }} as {{ dbt.type_string() }})
+            , cast({{ date_part("month", "paid_date") }} as {{ dbt.type_string() }})
+    )x
+
+    union all
+
+    select
+        date_field
+        , {{ dbt.concat(["year", dbt.right(dbt.concat(["'0'", "month"]), 2)]) }} as year_month
+        , distinct_count
+    from
+    (
+        select
+            'dispensing_date' as date_field
+            , cast({{ date_part("year", "dispensing_date") }} as {{ dbt.type_string() }}) as year
+            , cast({{ date_part("month", "dispensing_date") }} as {{ dbt.type_string() }}) as month
+            , count(distinct claim_id) as distinct_count
+        from {{ ref('core__pharmacy_claim') }}
+        group by
+            cast({{ date_part("year", "dispensing_date") }} as {{ dbt.type_string() }})
+            , cast({{ date_part("month", "dispensing_date") }} as {{ dbt.type_string() }})
+    )x
+
+    union all
+
+    select
+        date_field
+        , {{ dbt.concat(["year", dbt.right(dbt.concat(["'0'", "month"]), 2)]) }} as year_month
+        , distinct_count
+    from
+    (
+        select
+            'pharmacy paid_date' as date_field
+            , cast({{ date_part("year", "paid_date") }} as {{ dbt.type_string() }}) as year
+            , cast({{ date_part("month", "paid_date") }} as {{ dbt.type_string() }}) as month
+            , count(distinct claim_id) as distinct_count
+        from {{ ref('core__pharmacy_claim') }}
+        group by
+            cast({{ date_part("year", "paid_date") }} as {{ dbt.type_string() }})
+            , cast({{ date_part("month", "paid_date") }} as {{ dbt.type_string() }})
+    )x
+
+    union all
+
+    select
```

Comment:
I wonder if it makes sense to have member months in the analytics level and not atomic level? It feels like more of a concept since it'll be a pain to calculate based on the eligibility input_layer.

Reply:
@tom-tuva I ended up removing the query for member months.  I was using an old script I previously wrote for Insights and it had this column as the output.  The latest table design does not.